plugins/updater/src/lib.rs

```diff
@@ -163,9 +172,18 @@ impl Builder {
         self
     }
 
+    pub fn version_comparator<F: Fn(Version, RemoteRelease) -> bool + Send + Sync + 'static>(
```

Comment:
@amrbashir do you think that this name is good or should it be named `global_version_comparator` or `default_version_comparator`

Reply:
`default_version_comparator` sounds good